package.json

```diff
@@ -102,7 +102,7 @@
     "sass": "1.77.6",
     "sass-loader": "13.3.3",
     "swagger-client": "^3.17.0",
-    "swagger-ui": "^3.52.5",
+    "swagger-ui": "5.17.14",
```

Comment:
should we hardcode the version here? Not allow minor updates?

Reply:
Yeah, I don't know... @josemigallas seemed to prefer fixed versions for npm deps lately, just to avoid unexpected issues on version bumps.
I am not sure whether it's a good idea or not.

Reply:
I think that `package.json` is advisory. Once version is locked it will stay the same until updated.

So when you fix the version, I, as a reader, am thinking whether you know that future patch versions are broken or what was the reason for a fixed version.

While a normal version specification, locked to the major or minor version would indicate that we just don't want unexpected updates.

So not a blocking complaint but I think locking to the minor version is much better.